local/local.go

```diff
@@ -36,14 +36,13 @@ func SetIPForwarding() error {
 
 // SetIPForwardingUnix - sets the ipforwarding for linux
 func SetIPForwardingUnix() error {
-	x := []byte{0x01}
 	// ipv4
-	if err := os.WriteFile("/proc/sys/net/ipv4/ip_foward", x, os.ModePerm); err != nil {
+	if err := os.WriteFile("/proc/sys/net/ipv4/ip_forward", []byte("1"), os.ModePerm); err != nil {
```

Comment:
This change effectively makes `x := [49]` instead of `x := [1]`. Is this intended, and why?

Reply:
The other way give error Invalid Arg, the new way results in file containing 1